feature-tests/utils/utility.ts

```diff
@@ -22,3 +22,38 @@ export interface CurrentTimeDescriptor {
   milliseconds: number;
   seconds: number;
 }
+
+export type inputObjectForUpdatingItem = {
+  updatedAt: number;
+  appliedAt: number;
+  sentAt: number;
+  intervention: string;
+  blocked: boolean;
+  suspended: boolean;
+  resetPassword: boolean;
+  reproveIdentity: boolean;
+  history: string;
+  auditLevel: string;
+  isAccountDeleted?: boolean;
+  reprovedIdentityAt?: number;
+  resetPasswordAt?: number;
+  accountDeletedAt?: number;
+};
+
+export interface UserInformationFromTable {
+  pk: string;
+  updatedAt: number;
+  appliedAt: number;
+  sentAt: number;
+  intervention: string;
+  blocked: boolean;
+  suspended: boolean;
+  resetPassword: boolean;
+  reproveIdentity: boolean;
+  auditLevel: string;
```

Comment:
shouldn't Audit level be optional? Also, as an observation, isn't this interface very similar to the one above? Have you check if you can use one for both scenarios? :)

Reply:
Is it optional now, must of misremembered that, good catch, Yeah they are, just sloppy from me i think aha ill go and fix this, thank you!